### ISSUE TITLE
Beagle crashes on startup

### ISSUE THREAD

**Issue by truenicoco** [tigase/beagle-im#133]
Since a few days, I cannot useBeagleIM on my M1 macbook pro anymore. This is the crash report:

The crash report is too long for a github comment, but you can read it on [paste.sr.ht](https://paste.sr.ht/~nicoco/00ee7029afcee482fe125a280f7dc3241261a909)

**Comment by hantu85** [comment]
I've reviewed the crash log and it looks like it may be caused by an issue with a MUC bookmark or handling of a bookmark in Beagle IM (synchronization). To confirm that, and as a workaround, I would suggest disabling any network connection, starting BeagleIM, and in `Preferences` disabling the synchronization of group chats.

Would you be able to review your bookmarks and check if there is any MUC room bookmark with autojoin enabled and with a room jid without localpart? (no room name/just domain name?)

**Comment by truenicoco** [comment]
You nailed it, I had a bogus bookmark entry indeed (no @). It was caused by the component I develop and I definitely need to investigate how it happened.

Removing the bogus bookmark from Conversations fixed the issue. I guess it would be nice if such entries didn't prevent Beagle from starting.

**Comment by hantu85** [comment]
Yes, the cause of the issue will be fixed in the future version of Beagle IM. I just wanted to give you a way to possibly fix it for now and to confirm if that was the actual cause of the issue.
